runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -180,6 +182,18 @@ public void onWidgetLoaded(WidgetLoaded event)
 			case (WidgetID.THEATRE_OF_BLOOD_GROUP_ID):
 				eventType = "Theatre of Blood";
 				container = client.getItemContainer(InventoryID.THEATRE_OF_BLOOD_CHEST);
+
+				// Suppress reward chests where you receive an onion
+				if (container != null)
+				{
+					for (Item i : container.getItems())
```

Comment:
container.getItems().contains()? or is it array?

Reply:
Yeah it's an array (`Item[]`)

Reply:
Instead of doing this, just limit it to the region the reward chest is in maybe?